src/x86/rfxcodec_encode_dwt_shift_x86_sse2.asm

```diff
@@ -21,6 +22,29 @@
 
 %ifidn __OUTPUT_FORMAT__,elf
 section .note.GNU-stack noalloc noexec nowrite progbits
+%ifdef PIC
```

Comment:
No, we cannot require this, because it won’t be defined in non-PIC builds (global offset table is undefined). Furthermore, it’s a (slight) slowdown and removes one of the scarce CPU registers from application use.